020-LinuxFundamentals/Coach/README.md

```diff
@@ -0,0 +1,68 @@
+1. Challenge 07: **[Group and user management](../Coach/Solution-07.md)**
+	 - In this challenge you will learn about the creation of user and groups in a Linux environment.
+1. Challenge 08: **[Scripting](../Coach/Solution-08.md)**
+	 - Learn some basic stuff on shell scripting and the usage of some commands such as echo, cut, read and grep.
+1. Challenge 09: **[Disks, partitions and file systems](../Coach/Solution-09.md)**
+	 - You will be working with disks and partitions and learn about Linux filesystems and commands such as fdisk, mkfs and mount.
+1. Challenge 10: **[Logical Volume Mananager](../Coach/Solution-10.md)**
+	 - Discover about the Logical Volume Manager on Linux, and how to use commands such as pvcreate, vgcreate, lvrcreate, and more.
+1. Challenge 11: **[Package management](../Coach/Solution-11.md)**
+	 - Learn about package management and common activites such as update package distribution lists, install and uninstall packages.
+1. Challenge 12: **[Setting up a webserver](../Coach/Solution-12.md)**
+	 - In this challenge we will setting up a webserver and deploy a simple PHP application into it. The usage of SSL could be a plus.
+
+## Coach Prerequisites 
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to ensure students can access the contents of the \`/Student/Resources\` folder.
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.  
+
+## Azure Requirements
+
+This hack requires students to have access to an Azure subscription where they can create and consume Azure resources. These Azure requirements should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students.
+
+- For Challenge 01 an Azure subscription with contributor access will be required.
+- For all other challenges, at least a contributor access to a pre-created Ubuntu Linux 20.04 virtual machine will be required.
+- For the optional advanced challenge from Challenge 12, these are the requirements:
+	- A public IP attached to the virtual machine
+	- Access to the public IP of the virtual machine
+	- Access to the Azure App Service Domain to get a domain or a acess to the DNS management of an existent domain
+
+## Repository Contents
+
+- \`./Coach\`
```

Comment:
Remove all of the "\" escape characters but leave the back ticks in for the monospace font.